core/src/themes/ios/default.tokens.ts

```diff
@@ -12,10 +14,16 @@ export const defaultTheme: DefaultTheme = {
   palette: {
     light: lightTheme,
     dark: darkTheme,
+    highContrast: highContrastTheme,
+    highContrastDark: highContrastDarkTheme,
   },
 
   fontFamily: '-apple-system, BlinkMacSystemFont, "Helvetica Neue", "Roboto", sans-serif',
 
+  color: {
+    'overlay-bg': 'var(--ion-overlay-background-color, var(--ion-color-gray-100))', // Available only in iOS
```

Comment:
Created a functional color `overlay-bg` because it is being used by multiple components. It didn't make sense to create a new variable per component. However, I'm not opposed to making it per component.